free-2.md

```diff
@@ -58,12 +124,30 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Although designing the various GUIs for this project, especially the App,
+can be very time consuming and not directly related to the language aspects of
+this project, the GUIs can be mostly overlooked for this project. At the end
+of the day, the goal is to design a series of command line programs and
+simple macro scripts are easily extendable and callable from a GUI, not make
+the actual GUI. Although it would probably be nice to make a very simple
+computer GUI to demonstrate how the language could be extended and called
+from elsewhere, this could be kept very restricted to allow most of the
+development to be directly related to the language design. 
```

Comment:
I agree with what you say here. As a series of commands, this would be no fun at all to use, but could serve well as a 111 project.